aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -232,10 +232,29 @@
       "type": "string",
       "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
     },
+    "DomainAuthSecretArn": {
+      "type": "string",
+      "description": "The ARN for the Secrets Manager secret that contains the credentials for the user performing the domain join."
+    },
+    "DomainDnsIps": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "The IPv4 DNS IP addresses of your primary and secondary Active Directory domain controllers."
+    },
+    "DomainFqdn": {
+      "type": "string",
+      "description": "Specifies the fully qualified domain name of an Active Directory domain."
```

Comment:
Public docs are slightly different
`The fully qualified domain name (FQDN) of an Active Directory domain.`